pkgs/build-support/rust/build-rust-package/default.nix

```diff
@@ -117,7 +117,6 @@ stdenv.mkDerivation ((removeAttrs args [ "depsExtraArgs" "cargoUpdateHook" "carg
 
   nativeBuildInputs = nativeBuildInputs ++ [
     cacert
```

Comment:
iiuc cacert can be removed here as well?

Reply:
Good point, cacert shouldn't even have any "native" parts and would at most be in `buildInputs`. 

This has more potential for breakage though as I can imagine some builds might have depended on cacert before (without declaring it).  
If a build depends on something it doesn't declare, that's a bug though and it should be fixed by its maintainers. I think we can apply the pull principle here.

Reply:
Do you know any rust packages that do internet stuff and might depend on cacert implicitly?

Reply:
I've moved it to buildInputs. I can't gauge whether removing it is a good idea or not. If you think it can, please go ahead and remove it in another PR.